src/hotspot/cpu/aarch64/templateInterpreterGenerator_aarch64.cpp

```diff
@@ -1435,6 +1426,9 @@ address TemplateInterpreterGenerator::generate_native_entry(bool synchronized) {
     __ bind(no_oop);
   }
 
+  // Restore SP (drop native parameters area), to keep SP in sync with extended_sp in frame
+  // __ restore_sp_after_call();
```

Comment:
This seems to happen further down, around line 1530, but it's interleaved with popping the frame. Is there a reason to have the commented out call here?

Reply:
Ah, Nick got here before me. I also assume this needs to go.